File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){
    int alt = get_int("Height: ");
    int a = alt;
    int b = 0;

    for(int linha= 1 ;linha<=alt;linha++){//linha
        for(int esp = 1;esp<a;esp++){
            printf(" ");
        }
        b = linha;
        for(int c = 0;c<b;c++){
            printf("#");
        }
        printf(" ");
        for(int c = 0;c<b;c++){
            printf("#");
        }
        printf("\n");
        a--;

    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while(True):
    altura = get_int("Height: ")
    if((altura<1)or(altura>=9)):
        continue
    if(type(altura) is int):
        break
    else:
        continue

for x in range(1,altura+1):
    print(" "*(altura-x)+"#"*(x)+"  "+"#"*(x))
